Sprint-2/stretch/count-words.js

```diff
@@ -26,3 +26,38 @@
 
 3. Order the results to find out which word is the most common in the input
 */
+// Your countWords function
+function countWords(string) {
+  const punctuation = [".", ",", "!", "?", '"'];
+  const counter = {};
+  const words = string.split(" ");
+
+  const cleanWords = words.map((word) => {
+    let cleanWord = "";
+    for (let char of word) {
+      if (!punctuation.includes(char.toLowerCase())) {
```

Comment:
Thank you for pointing that out! You're absolutely correct—`char.toLowerCase()` is unnecessary here because punctuation marks don't have case variations. I'll simplify the condition to:

```javascript
if (!punctuation.includes(char)) {